recipes/usrsctp/all/conanfile.py

```diff
@@ -0,0 +1,62 @@
+from conans import ConanFile, CMake, tools, AutoToolsBuildEnvironment
+from shutil import copyfile
+import os
+
+
+class ConanRecipe(ConanFile):
+    name = "usrsctp"
+    description = "SCTP user-land implementation (usrsctp)"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/sctplab/usrsctp"
+    license = "BSD 3"
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "programs": [True, False],
+        "fuzzer": [True, False],
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True,
+        "programs": False,
+        "fuzzer": False,
+    }
+    generators = "cmake"
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+        del self.settings.compiler.cxx
+        del self.settings.compiler.cppstd
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        extracted_dir = self.name + "-" + self.version
+        os.rename(extracted_dir, self._source_subfolder)
+
+    def _configure_cmake(self):
+        cmake = CMake(self)
+        cmake.definitions["sctp_build_programs"] = self.options.programs
+        cmake.definitions["sctp_build_fuzzer"] = self.options.fuzzer
+        cmake.configure(source_folder=self._source_subfolder)
```

Comment:
```suggestion
        cmake.configure()
```

If you point to source_subfolder, the cmake wrapper will be ignored.